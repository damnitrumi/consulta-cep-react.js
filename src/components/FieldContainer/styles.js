import styled, { css } from "styled-components";

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-direction: column;
    gap: ${theme.spacings.tiny};
    margin-bottom: ${theme.spacings.medium};
  `}
`;
