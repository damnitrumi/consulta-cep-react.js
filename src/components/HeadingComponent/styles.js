import styled, { css } from "styled-components";

export const Container = styled.h1`
  ${({ theme }) => css`
    max-width: auto;
    margin: ${theme.spacings.tiny} 0;
    color: red;
  `}
`;
