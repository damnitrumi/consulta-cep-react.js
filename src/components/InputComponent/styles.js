import styled, { css } from "styled-components";

export const Container = styled.input`
  ${({ theme }) => css`
    padding: ${theme.spacings.tiny};
    outline-color: #0fccff;
    font-size: ${theme.font.sizes.small};
  `}
`;
