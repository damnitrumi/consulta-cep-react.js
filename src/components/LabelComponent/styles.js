import styled, { css } from "styled-components";

export const Container = styled.label`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.medium};
  `}
`;
