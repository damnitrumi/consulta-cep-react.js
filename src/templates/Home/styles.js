import styled, { css } from "styled-components";

export const Container = styled.div`
  ${({ theme }) => css`
    padding: ${theme.spacings.small};
  `}
`;
