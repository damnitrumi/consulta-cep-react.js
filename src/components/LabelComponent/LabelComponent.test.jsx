import { screen } from "@testing-library/react";
import { renderTheme } from "../../styles/render-theme";
import { LabelComponent } from ".";

describe("<LabelComponent />", () => {
  it("should render LabelComponent", () => {
    renderTheme(<LabelComponent forId="teste" />);
    const label = screen.getByText(/Teste/i);
    expect(label).toHaveAttribute("for", "teste");
  });

  it("should render LabelComponent with a small id", () => {
    renderTheme(<LabelComponent forId="cep" />);
    const label = screen.getByText(/cep/i);
    expect(label).toHaveAttribute("for", "cep");
  });

  it("should render match snapshot", () => {
    renderTheme(<LabelComponent forId="teste" />);
    const label = screen.getByText(/Teste/i);
    expect(label).toMatchSnapshot();
  });
});
