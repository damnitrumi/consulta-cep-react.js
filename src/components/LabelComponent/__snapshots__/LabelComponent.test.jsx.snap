// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LabelComponent /> should render match snapshot 1`] = `
.c0 {
  font-size: 2.4rem;
}

<label
  class="c0"
  for="teste"
>
  Teste
</label>
`;
